control/tests/test_delivery_cielo.py

```diff
@@ -1,18 +1,224 @@
+
+    def assert_request(self, received_request, expected_request):
+        """
+        Verify that `received_request` matches `expected_request`
+        """
+        self.assertEqual(received_request.method, expected_request['method'])
+        self.assertEqual(received_request.url, expected_request['url'])
+        self.assertEqual(received_request.body, expected_request['body'])
+
+    @patch('control.veda_deliver_cielo.VALAPICall._AUTH', PropertyMock(return_value=lambda: CONFIG_DATA))
+    @responses.activate
+    def test_transcript_flow(self):
+        """
+        Verify cielo24 transcription flow
+        """
+        job_id = '000-111-222'
+
+        cielo24 = Cielo24Transcript(
+            video=self.video,
+            video_transcript_preferences=self.video_transcript_preferences
+        )
+
+        responses.add(responses.POST, CONFIG_DATA['val_token_url'], '{"access_token": "1234567890"}', status=200)
+        responses.add(responses.PATCH, CONFIG_DATA['val_video_transcript_status_url'], status=200)
+        responses.add(
+            responses.GET,
+            self.cielo24_url(cielo24, cielo24.cielo24_new_job),
+            body={'JobId': job_id},
+            status=200
         )
-    output = x.perform_transcription()
-    print output
+        responses.add(
+            responses.GET,
+            self.cielo24_url(cielo24, cielo24.cielo24_add_media),
+            body={'TaskId': '000-000-111'},
+            status=200
+        )
+        responses.add(
+            responses.GET,
+            self.cielo24_url(cielo24, cielo24.cielo24_perform_transcription),
+            body={'TaskId': '000-000-000'},
+            status=200
+        )
+
+        cielo24.start_transcription_flow()
+
+        # Total of 8 HTTP requests are made
+        # 2 requests for val
+        # 3 cielo24 requests for first language(en)
+        # 3 cielo24 requests for second language(ur)
+        self.assertEqual(len(responses.calls), 8)
+
+        # pylint: disable=line-too-long
+        expected_data = [
+            {
+                'url': 'https://api.cielo24.com/api/job/new?api_token=cielo24_api_key&job_name=&language=en&v=1',
+                'body': None,
+                'method': 'GET'
+            },
+            {
+                'url': 'https://api.cielo24.com/api/job/add_media?media_url=intro_to_db_DTH.mp4&api_token=cielo24_api_key&job_id=000-111-222&v=1',
+                'body': None,
+                'method': 'GET'
+            },
+            {
+                'url': 'https://api.cielo24.com/api/job/perform_transcription?transcription_fidelity=PROFESSIONAL&job_id=000-111-222&v=1&priority=PRIORITY&api_token=cielo24_api_key&callback_url=%252Fcielo24%252Ftranscript_completed%253Fjob_id%253D000-111-222%2526lang_code%253D{}%2526org%253DMAx%2526video_id%253D12345&target_language={}',
+                'body': None,
+                'method': 'GET'
+            }
+        ]
+
+        received_request_index = 2
```

Comment:
yes, don't want to check val requests, those are already verified in `test_transcripts.py`